src/diffusers/pipelines/stable_diffusion/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 import PIL
 from PIL import Image
+from transformers import BatchEncoding
```

Comment:
Could you please move this under line `34` (below `is_transformers_available()`) ? Otherwise this breaks the init as transformers is not a hard dependency